src/devices/Adg731/samples/README.md

```diff
@@ -0,0 +1,102 @@
+# ADG731 - Samples
+
+## Required hardware
+* [Analog Devices ADG731](https://www.analog.com/media/en/technical-documentation/data-sheets/ADG725_731.pdf)
+* Any supported IoT device with an SPI bus
```

Comment:
actually that's not fully true, you it will probably work on any device with GPIO as well since we support software SPI, also will work with any PC which has FT4222 USB dongle which exposes SPI through one of our bindings. I'd skip the required board section per device since this can easily get out of date in the future